lib/airtap.js

```diff
@@ -6,9 +6,9 @@ var EventEmitter = require('events').EventEmitter
 
 var createControlServer = require('./control-app')
 var setup_test_instance = require('./setup')
-var SauceBrowser = require('./SauceBrowser')
-var PhantomBrowser = require('./PhantomBrowser')
-var Electron = require('./Electron')
+var SauceBrowser = require('./sauce-browserlist')
+var PhantomBrowser = require('./phantom-browserlist')
+var Electron = require('./electron-browserlist')
```

Comment:
`sauce-browser`
`phantom-browser`
`electron-browser`

Reply:
Are you saying the variable name should follow the above pattern as well? Or are you saying the `require` statements are wrong? Sorry I'm a bit confused 😓

Reply:
The require statements are wrong. Those wouldn't be valid variables names :wink:

Reply:
Maybe also rename `Electron` to `ElectronBrowser` for symmetry.